src/Ocelot/Configuration/File/FileAuthenticationOptions.cs

```diff
@@ -1,28 +1,31 @@
 ﻿namespace Ocelot.Configuration.File
 {
-    public class FileAuthenticationOptions
+    public sealed class FileAuthenticationOptions
     {
         public FileAuthenticationOptions()
         {
             AllowedScopes = new List<string>();
+            AuthenticationProviderKeys = Array.Empty<string>();
         }
 
         public FileAuthenticationOptions(FileAuthenticationOptions from)
         {
             AllowedScopes = new(from.AllowedScopes);
             AuthenticationProviderKey = from.AuthenticationProviderKey;
+            AuthenticationProviderKeys = from.AuthenticationProviderKeys;
         }
 
-        public string AuthenticationProviderKey { get; set; }
         public List<string> AllowedScopes { get; set; }
+        public string AuthenticationProviderKey { get; set; }
+        public string[] AuthenticationProviderKeys { get; set; }
 
-        public override string ToString()
-        {
-            var sb = new StringBuilder();
-            sb.Append($"{nameof(AuthenticationProviderKey)}:{AuthenticationProviderKey},{nameof(AllowedScopes)}:[");
-            sb.AppendJoin(',', AllowedScopes);
-            sb.Append(']');
-            return sb.ToString();
-        }
+        public override string ToString() => new StringBuilder()
+            .Append($"{nameof(AuthenticationProviderKey)}:{AuthenticationProviderKey},{nameof(AuthenticationProviderKeys)}:[")
+            .AppendJoin(',', AuthenticationProviderKeys)
+            .Append("],")
+            .Append($"{nameof(AllowedScopes)}:[")
+            .AppendJoin(',', AllowedScopes)
+            .Append(']')
+            .ToString();

```

Comment:
```suggestion
        public override string ToString() =>new StringBuilder()
        .Append($"{nameof(AuthenticationProviderKey)}:{AuthenticationProviderKey},")
        .Append($"{nameof(AuthenticationProviderKeys)}:[{string.Join(',', AuthenticationProviderKeys)}],")
        .Append($"{nameof(AllowedScopes)}:[{string.Join(',', AllowedScopes)}]")
        .ToString();

```

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1870/commits/27869622919503ba4749c6de271a02cb6df08ab1 ✔️